Gemfile

```diff
@@ -16,8 +16,11 @@ gem 'facter', '>= 1.6.2'
 group :test, :development do
   gem 'rspec', '~> 2.10.0'
   gem 'mocha', '~> 0.10.5'
+  gem 'augeas'
```

Comment:
This a sad, long-standing issue… https://github.com/hercules-team/ruby-augeas/pull/2